training/src/components/index.js

```diff
@@ -1 +1,5 @@
-export { TextField } from './TextField';
+export { TextField } from './TextField/index';
```

Comment:
export { TextField } from './TextField/index';
export { Slider } from './slider/index';
export { ButtonField } from './Button/index';
export { SelectField } from './SelectField';
export { RadioGroup } from './RadioGroup/index';

instead of this you can simply use 

export { TextField } from './TextField';
export { Slider } from './slider;
export { ButtonField } from './Button';
export { SelectField } from './SelectField';
export { RadioGroup } from './RadioGroup'